app/Resources/views/full/blog_post.html.twig

```diff
@@ -100,8 +100,8 @@
                                     <div class="col-xs-3"></div>
                                     <div class="col-xs-6 center">
                                         <ul class="list-inline-new">
-                                            <li><a href="//www.facebook.com/sharer/sharer.php?u={{ app.request.uri }}{{ path(location) }}" target="_blank"><img src="{{ asset('assets/images/share_facebook.svg') }}" /></a></li>
-                                            <li><a href="//twitter.com/home?status={{ app.request.uri }}{{ path(location) }}" target="_blank"><img src="{{ asset('assets/images/share_twitter.svg') }}" /></a></li>
+                                            <li><a href="{% if location.id is empty %}#"{% else %}//www.facebook.com/sharer/sharer.php?u={{ app.request.uri }}{{ path(location) }}" target="_blank"{% endif %}><img src="{{ asset('assets/images/share_facebook.svg') }}" /></a></li>
```

Comment:
That's the way to go. A bit better would be `{% if not location.is_draft %}`.

During the preview of the first version of any content, we don't have a location yet. A draft one is built for the preview, that for instance has the correct path so that breadcrumbs or link to parent can be generated.